client/src/main/java/agolf/game/GameBackgroundCanvas.java

```diff
@@ -60,28 +44,20 @@ public void paint(Graphics g) {
     public void update(Graphics g) {
         if (this.image == null) {
             g.setColor(GolfGameFrame.colourGameBackground);
-            g.fillRect(0, 0, 735, 375);
+            g.fillRect(0, 0, this.trackWidth, this.trackHeight);
         } else {
             g.drawImage(this.image, 0, 0, this);
         }
     }
 
-    protected String[] generateTrackInformation() {
-        return new String[] {this.trackAuthor, this.trackName, this.trackFirstBest, this.trackLastBest};
-    }
-
-    protected int[][] generateTrackStatistics() {
-        return new int[][] {this.trackStats, this.trackRatings};
-    }
-
     // this useless func is called when we get start packet
     // iniatilize variables
     // draws map as grass
     // this.gameCanvas.createMap(16777216);
     // 16777216 == grass
-    protected void createMap(int var1) {
+    protected void createMap(int tile) {
```

Comment:
It looks to me that all this function does is initialize a map as all grass-tiles so this function could maybe be renamed for clarity to something like `initializeMap`.

Also, this function is only ever called with `tile` as 16777216, so the parameter could be eliminated and code simplified a bit.